runtime-tests/src/test/java/com/dylibso/chicory/testing/TestModule.java

```diff
@@ -60,8 +64,27 @@ public TestModule(Module module) {
         this.module = module;
     }
 
+    private static Map<OpCode, Machine.OpImpl> additionalOpcodes;
+
+    static {
+        // TODO: discuss - either we use reflection or we need to restructure the project modules
```

Comment:
An alternative approach would be to fully duplicate the `runtime-tests` module, the downside is that needs a bit of fiddling with profiles in Maven.